.yamllint.yml

```diff
@@ -0,0 +1,15 @@
+extends: default
```

Comment:
noooo another top level file

(╯°□°)╯︵ ┻━┻

fine..

Reply:
Hehe new you would hate that :-D Sorry I think that's really the standard place for it though...